guide/sections/part3/sensor-centres.adoc

```diff
@@ -5,7 +5,7 @@ Centres can provide external feedback on WIS 2.0 functions to trigger corrective
 
 ==== 3.3.1 Overview
 
-As described in the Manual of WIS, Volume II, WIS 2.0 is a collective, distributed data exchange system that requires monitoring. 
+As described in the Manual on WIS, Volume II, WIS 2.0 is a collective, distributed data exchange system that requires monitoring. 
 
 All WIS 2.0 components have defined metrics and conditions on those metrics to trigger alerts.
 
```

Comment:
The "Sensor Centre" is a generic term to describe additional components and tool to get a particular assessment on:
- the quality and performance of the Global Services
- the conformance of the products exchanged with the agreed regulations (eg. for RBON)
- ...

Each kind of Sensor Centre will result in a set of metrics that will be used to measure the performance.

Reply:
Updated.